code/obj/item/gun/energy.dm

```diff
@@ -298,7 +298,7 @@ TYPEINFO(/obj/item/gun/energy/antique)
 			if(180 to INFINITY)
 				//not good enough to be functional
 				return 0
-		switch(src.myCoil.material.getProperty("electrical") + ((src.myCoil.material.getMaterialFlags() & MATERIAL_ENERGY) ? 2 : 0))
+		switch(src.myCoil.material.getProperty("\"electrical") + ((src.myCoil.material.getMaterialFlags() & MATERIAL_ENERGY) ? 2 : 0))
```

Comment:
??

Reply:
I uh... hm. It worked in testing? I legitimately dont remember doing that. Fixed